Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiDisplay.h

```diff
@@ -92,7 +92,12 @@ namespace AzToolsFramework::ViewportUi::Internal
         bool IsViewportUiElementVisible(ViewportUiElementId elementId);
 
         void CreateViewportBorder(const AZStd::string& borderTitle, AZStd::optional<ViewportUiBackButtonCallback> backButtonCallback);
+        void ChangeViewportBorderText(const AZStd::string& borderTitle);
         void RemoveViewportBorder();
+        const bool ViewportBorderVisible()
```

Comment:
update: This should be be returning a `const bool`, the function should be marked `const`. So this should be `bool ViewportBorderVisible() bool` (it would also be good to move the implementation out of line to keep this interface cleaner)

```
inline bool ViewportUiDisplay::ViewportBorderVisible() const
{
    ...
}
```